knack/commands.py

```diff
@@ -276,10 +276,10 @@ def _get_op_handler(operation):
                 op = getattr(op, part)
             if isinstance(op, types.FunctionType):
                 return op
-            # MethodType
+            # op as types.MethodType
             return op.__func__
-        except (ValueError, AttributeError):
-            raise ValueError("The operation '{}' is invalid.".format(operation))
+        except (ValueError, AttributeError) as ex:
+            raise ValueError("The operation '{}' is invalid.".format(operation)) from ex
```

Comment:
Use Python's built-in Exception Chaining as recommended by [W0707](http://pylint.pycqa.org/en/latest/technical_reference/features.html#exceptions-checker-messages):


> raise-missing-from (W0707):
> Consider explicitly re-raising using the 'from' keyword Python 3's exception chaining means it shows the traceback of the current exception, but also the original exception. Not using raise from makes the traceback inaccurate, because the message implies there is a bug in the exception-handling code itself, which is a separate situation than wrapping an exception.



- Exception Chaining: https://docs.python.org/3.9/tutorial/errors.html#exception-chaining
- Built-in Exceptions: https://docs.python.org/3.9/library/exceptions.html#bltin-exceptions